src/applications/vaos/tests/appointment-list/components/ConfirmedAppointmentDetailsPage/index.unit.spec.jsx

```diff
@@ -1156,7 +1163,7 @@ describe('VAOS <ConfirmedAppointmentDetailsPage> with VAOS service', () => {
     };
     // When fetching the specific appointment id
     const url = '/va/1234';
-    const futureDate = moment(getTimezoneTestDate());
+    const futureDate = moment.utc();
```

Comment:
The test was failing for me every time I ran the file. The form was not putting the link to add to the calendar. I was just setting this to the value that seemed to be the pattern in the rest of the tests ( see line 1062). I'm wondering if it fails for me since I am on the pacific time zone?